### ISSUE TITLE
field_security in kibana_role not working

### ISSUE THREAD

**Issue by Dalmako** [disaster37/terraform-provider-kibana#10]
I'm trying to use the field_security feature in the kibana_role but it's not working. 

Example:
```
resource kibana_role "my_kibana_role" {
  name = "my_kibana_role"
  elasticsearch {
    indices {
      names = ["*"]
      privileges = ["read"]
      field_security = jsonencode({
        "grant": ["my_field"]
      })
    }
  }
}
```

I also tried this...
```
resource kibana_role "my_kibana_role" {
  name = "my_kibana_role"
  elasticsearch {
    indices {
      names = ["*"]
      privileges = ["read"]
      field_security = "{\"grant\": [\"my_field\"]}"
  }
}
```

I think the problems is [here](https://github.com/disaster37/terraform-provider-kibana/blob/master/kb/resource_kibana_role.go#L297) after calling to the optionalInterfaceJSON few lines above when you are returning a json.RawMessage.

Could you take a look?

**Comment by disaster37** [comment]
Hi,

can you test with
```
resource kibana_role "my_kibana_role" {
  name = "my_kibana_role"
  elasticsearch {
    indices {
      names = ["*"]
      privileges = ["read"]
      field_security = <<EOF
{
   "grant": ["my_field"]
}
EOF
  }
}
```

If not working, what is error message, and log appear in kibana logs ?

**Comment by disaster37** [comment]
Fix on release [v7.12.9](https://github.com/disaster37/terraform-provider-kibana/releases/tag/v7.12.9)
The code used to prove it: https://github.com/disaster37/terraform-provider-kibana/blob/7.x/tests/kibana_role/terraform.tf
